crates/rpc/rpc/src/eth/helpers/types.rs

```diff
@@ -1,72 +1,40 @@
 //! L1 `eth` API types.
 
-use alloy_consensus::TxEnvelope;
-use alloy_network::{Ethereum, Network};
-use alloy_rpc_types::TransactionRequest;
-use alloy_rpc_types_eth::{Transaction, TransactionInfo};
+use alloy_network::Ethereum;
 use reth_ethereum_primitives::EthPrimitives;
-use reth_primitives_traits::{Recovered, TxTy};
 use reth_rpc_eth_api::EthApiTypes;
 use reth_rpc_eth_types::EthApiError;
-use reth_rpc_types_compat::TransactionCompat;
+use reth_rpc_types_compat::RpcTransactionConverter;
+use std::fmt::Debug;
 
 /// A standalone [`EthApiTypes`] implementation for Ethereum.
-#[derive(Debug, Clone, Copy, Default)]
-pub struct EthereumEthApiTypes(EthTxBuilder);
+#[derive(Debug, Clone, Default)]
+pub struct EthereumEthApiTypes(RpcTransactionConverter<EthPrimitives, Ethereum, EthApiError>);
```

Comment:
Should we also remove `EthereumEthApiTypes`? It does not seem to be used anywhere, but is `pub`.

cc @mattsse